### DISCUSSION TITLE
Text being deleted in GitHub editor

### DISCUSSION THREAD

**Discussion by MoritzLost** [community/community#58737] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I'm experiencing an issue where I type something into the editor in the new code view, and it's deleted a couple of seconds later. See the following video:

https://github.com/community/community/assets/10146880/2d9d3282-cd74-4921-9159-256f3bb6e00a

This happens periodically, and the issue has persisted after a reload. This is making the code editing on GitHub impossible to use.

**Comment by Cbodfield** [comment, marked as answer]
@MoritzLost I apologize for the friction and frustration while editing but thank you for reporting the problem.

Have you noticed it happening more often for any particular file types? (ie - more on markdown like your video)

Is this happening more often when you are creating a new file or editing an existing one?

I'm looking into it on my end but any additional info or ways to reproduce are appreciated!

**Reply by Cbodfield** [reply]
@MoritzLost This problem should be resolved now. If it's not, please let me know.

**Reply by MoritzLost** [reply]
@Cbodfield I can't reproduce it now, so looks like your fix worked. Thanks!

**Comment by waynebitbean** [comment]
I just changed the name of the file at the top of the page after typing out an entire Markdown document and it erased everything.

**Comment by Tyrannas** [comment]
I have the same issue on markdown files, happened a couple of files when editing online.
